# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1287384081 14400
#      Mon Oct 18 02:41:21 2010 -0400
# Node ID 36c18286a61bce14104cb60c3786ca739eba129b
# Parent  a6ab46b5926fdd90be9027710fb5aa03dfae8f5d
__errplot__.m: remove stray debugging statement

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-10-18  John W. Eaton  <jwe@octave.org>
+
+	* plot/private/__errplot__.m: Remove debugging statement from
+	previous change.
+
 2010-10-17  John W. Eaton  <jwe@octave.org>
 
 	* miscellaneous/edit.m: Use version 3 in GPL notice.
 
 	* @ftp/ascii.m, @ftp/binary.m, @ftp/cd.m, @ftp/close.m,
 	@ftp/delete.m, @ftp/dir.m, @ftp/display.m, @ftp/ftp.m,
 	@ftp/loadobj.m, @ftp/mget.m, @ftp/mkdir.m, @ftp/mput.m,
 	@ftp/rename.m, @ftp/rmdir.m, @ftp/saveobj.m,
diff --git a/scripts/plot/private/__errplot__.m b/scripts/plot/private/__errplot__.m
--- a/scripts/plot/private/__errplot__.m
+++ b/scripts/plot/private/__errplot__.m
@@ -29,18 +29,16 @@
 function h = __errplot__ (fstr, p, varargin)
 
   if (nargin < 4 || nargin > 8) # at least two data arguments needed
     print_usage ();
   endif
 
   [fmt, key] = __pltopt__ ("__errplot__", fstr);
 
-  fmt
-
   [len, nplots] = size (varargin{1});
   h = [];
 
   for i = 1:nplots
     ## Set the plot type based on linestyle.
 
     if (strcmp (fmt.errorstyle, "~"))
       ifmt = "yerr";
